test/e2e/integreatly_test.go

```diff
@@ -115,11 +134,54 @@ func integreatlyWorkshopTest(t *testing.T, f *framework.Framework, ctx *framewor
 			break
 		}
 	}
```

Comment:
I think we need to check err here before continuing:

```
if err != nil {
    return err
}
```

Reply:
Looks to me like we break out of the loop if hit an error, but never check it before continuing on to the next part https://github.com/integr8ly/integreatly-operator/blob/fbfb5fe39ade58fb68c621a152c2ddf9ca4a02db/test/e2e/integreatly_test.go#L131-L139

Should we not be returning err at that point if we have identified one of the product operators isn't running?